src/groups/mqb/mqbnet/mqbnet_clusteractivenodemanager.h

```diff
@@ -258,6 +258,13 @@ class ClusterActiveNodeManager {
     // Pointer to the currently active node
     // and its context.
 
+    bool d_ignoreDataCenter;
+    // If true, remove the data center
+    // requirement when selecting active
+    // node. Set to true when the cluster
+    // does not have any nodes in the
+    // current machine's data center.
+
```

Comment:
While this does _work_, it's not elegant. The bool below claims to do what this bool is doing, but the bool below actually affects much more (as noted earlier in the discussion). I might try to do some more refactoring here to clean this up a bit more.